src/utils/init.js

```diff
@@ -158,8 +159,12 @@ function validateStep(formKey, trigger = true) {
   return callback(trigger);
 }
 
-function setAuthClient(client) {
+function setAuthClient(client, triggerLogin = true) {
   initState.authClient = client;
+  // NEED to handle stytch vs firebase - which emailLogin to call
+  // not sure on the best way to get session.integrations info here
+  // Is it saved somewhere I'm overlooking? should we save it to the client or init?
+  if (triggerLogin) emailLogin(defaultClient);
```

Comment:
// NEED to handle stytch vs firebase - which emailLogin to call
  // not sure on the best way to get session.integrations info here
  // Is it saved somewhere I'm overlooking? should we save it to the client or init?